docs/source/collect/certificates.md

```diff
@@ -0,0 +1,105 @@
+---
+title: TLS Certificates
+description: Collect Certificate Chain information from Kubernetes ConfigMaps and Secrets in collected output
+---
+
+The `certificates` collector can be used to gather information about the TLS certificates from Kubernetes ConfigMaps and Secrets. This collector can be used multiple times, referencing different Secrets and ConfigMaps.
+
+## Parameters
```

Comment:
We are missing documentation for `secrets` and `configMap` parameters. These can be documented together since they share the same parameters. I suggest following how [runPod's imagePullSecret](https://troubleshoot.sh/docs/collect/run-pod/#imagepullsecret-optional) parameter has been documented. Note the bullet indentations to indicate hierarchy.